common/lib/xmodule/xmodule/library_tools.py

```diff
@@ -55,74 +53,18 @@ def update_children(self, dest_block, user_id, update_db=True):
         This method will update dest_block's 'source_libraries' field to store
         the version number of the libraries used, so we easily determine if
         dest_block is up to date or not.
-
-        If update_db is True (default), this will explicitly persist the changes
-        to the modulestore by calling update_item(). Only set update_db False if
-        you know for sure that dest_block is about to be saved to the modulestore
-        anyways. Otherwise, orphaned blocks may be created.
         """
-        root_children = []
+        new_libraries = []
+        source_blocks = []
+        for library_key, dummy in dest_block.source_libraries:
```

Comment:
I believe our standard for unused vars is `__` but I don't care as long as you're not getting pylint errors. (I think pylint ignores vars starting w/ `_`)